src/server/local-auth-helpers.js

```diff
@@ -30,11 +30,13 @@ const login = async ({
   uuidMatch
 }) => {
   if (existingUser.length === 0) {
-    return [null, false, errorMessages.invalidCredentials]
+    throw new Error(errorMessages.invalidCredentials)
   }
 
-  // Verify UUID validity
-  if (nextUrl) validUuid(nextUrl, uuidMatch)
+  // Verify UUID validity when an existing user is invited to a new
+  // campaign, e.g. dynamic assignament.
+  // If there is an error, it will be caught on local strategy invocation
+  if (nextUrl) await validUuid(nextUrl, uuidMatch)
```

Comment:
I like the new structure with throwing errors -- it's much easier to read and understand.

I'm still not clear if we actually need to validate this on login -- I know there will *be* a nextUrl with a uuid that we could test, but after login, isn't there another GraphQL api call to 'join' the org (in src/containers/JoinTeam.jsx)?  I suppose it doesn't hurt here, but I think it might make the logic a bit more confusing.